src/ace.js

```diff
@@ -1,17 +1,16 @@
 /**
  * The main class required to set up an Ace instance in the browser.
  *
- * @class Ace
```

Comment:
currently we allow both `import * as ace from "ace-code"` `import {ace} from "ace-code"`, i wonder if we can somehow align the namespace with this